specification/columns/tags.md

```diff
@@ -9,7 +9,8 @@ The Tags column adheres to the following requirements:
 * The Tags column MUST contain user-defined and provider-defined tags.
 * The Tags column MUST only contain finalized tags.
 * The Tags column MUST be in [Key-Value Format](#key-valueformat).
-* A Tag key without a specified value MUST have its tag value set to null.
+* A Tag key that supports a corresponding, non-null value tied to a resource SHOULD be included.
```

Comment:
I'm mostly good with line 12. It gives freedom to the cloud provider to decide what tags to include. Here is a suggested rewording to make it more clear the tag should have a non-null value and what should be included:

> A Tag key-value pair where the value is non-null for a given resource SHOULD be included in the tags column.

This also may raise the question to the reader "What about tag keys that have null values?". We could add a line that maintains the freedom for cloud providers:

> A Tag key-value pair where the value is null for a given resource MAY be included in the tags column, depending on the provider's tag finalization process.

Reply:
I like both but just without `-value pair` because it is defined as a KeyValueFormat attribute already.

Does this edits work? @AWS-ZachErdman 

```
A Tag key where its value is non-null for a given resource SHOULD be included in the tags column.
A Tag key where its value is null for a given resource MAY be included in the tags column, depending on the provider's tag finalization process.
```

Reply:
looks good to me!